### DISCUSSION TITLE
Welcome to lsyncd Discussions!

### DISCUSSION THREAD

**Discussion by poelzi** [axkibe/lsyncd#698] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions or share your problems.
  * Share ideas and configs
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

This Board replaces the google group.

**Comment by DangerD1024** [comment]
Please help me with exclude file
```
sync {
  default.rsync,
  source="/home/user/",
  target="user@host.com:/home/user/",
  delay           = 5,
  delete = false,
  excludeFrom = "/home/user/lsyncd.exclude2",
...
```

Lsyncd doesn't want to exclude files from exclude file, what i did wrong?

lsyncd.exclude2:

```
/home/user/lsyncd.exclude2
/home/user/file.txt
lsyncd.exclude2
file.txt
```

Both doesn't work...
What file paths i should write there?

Version: 2.3.1
